### PR TITLE
Adding OSM India FB Page

### PR DESCRIPTION
**Pull request by jinalfoflia**
Adding the OpenStreetMap India Facebook Page

cc: @bhousel

### CODE REVIEW

File: features/asia/india/india_full.geojson [hunk 1]
```diff
@@ -0,0 +1,148 @@
+{
+  "type": "Feature",
+  "id": "india_full",
+  "properties": {"name": "India"},
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
```
Comment (bhousel) [inline features/asia/india/india_full.geojson]: looks like MultiPolygon needs an additional nesting of coordinate array rings (add `[` )

File: features/asia/india/india_full.geojson [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        [74.405929, 31.692639],
+        [75.258642, 32.271105],
+        [74.451559, 32.7649],
+        [74.104294, 33.441473],
+        [73.749948, 34.317699],
+        [74.240203, 34.748887],
+        [75.757061, 34.504923],
+        [76.871722, 34.653544],
+        [77.837451, 35.49401]
+      ]
```
Comment (bhousel) [inline features/asia/india/india_full.geojson]: and a `]` here too!

**Comment by jinalfoflia** [comment]
Thank you for looking into this @bhousel, have added the extra nesting for multipolygon. Let me know if there are any other changes, thank you.

**Comment by bhousel** [comment]
Thanks @jinalfoflia !

**Pull request merged**